docs/pages/bare/installing-updates.mdx

```diff
@@ -26,22 +26,62 @@ Once installation is complete, apply the changes from the following diffs to con
 
 ## Configuration in JavaScript and JSON
 
-You'll need to modify **index.js** to import `expo-asset` early in your app, to be able to update assets with updates.
-
-```diff index.js
-+ import 'expo-asset';
-import { registerRootComponent } from 'expo';
-
-import App from './App';
-
-// registerRootComponent calls AppRegistry.registerComponent('main', () => App);
-// It also ensures that whether you load the app in Expo Go or in a native build,
-// the environment is set up appropriately
-registerRootComponent(App);
+Modify the `expo` section of **app.json**. (If you originally created your app using `npx react-native init`, you will need to add this section.)
+The changes below add the updates URL and a request header to the Expo configuration.
+
+```diff app.json
+ {
+   "name": "MyApp",
+-  "displayName": "MyApp"
++  "displayName": "MyApp",
++  "expo": {
++    "name": "MyApp",
++    "slug": "MyApp",
++    "ios": {
++      "bundleIdentifier": "com.MyApp"
++    },
++    "android": {
++      "package": "com.MyApp"
++    },
++    "runtimeVersion": "1.0.0",
++    "updates": {
++      "requestHeaders": {
++        "expo-channel-name": "your-channel-name"
++      },
++      "url": "https://exp.host/@my-expo-username/my-app"
```

Comment:
this should be a modern updates URL (for `expo-channel-name` header to make sense)

Reply:
So there are two ways to handle this:
1) Show what you would do if you are using a custom update server;
2) Show what you would do for EAS update.

For (2) you wouldn't even add these, you would just run `eas init` and `eas update:configure` and let EAS set these.

Reply:
@wschurman I got the custom update server flow to work with `"url": "http://localhost:3000/api/manifest"`. After prebuild, if you started with a vanilla RN CLI project and not an Expo project, you have to remove `expo-splash-screen` dependency, and then manually add the `Expo.plist` file to the Xcode project. No other changes were needed. I believe if you use a project started with `create-expo-app` then these changes would not be necessary. Will test Android to see if there are any hand edits needed to make that work.

Reply:
I've modified the URL and made other related changes.